src/command_modules/azure-cli-datalake/azure/cli/command_modules/datalake/store/_validators.py

```diff
@@ -0,0 +1,38 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+from azure.cli.core.commands.client_factory import get_mgmt_service_client
+from azure.mgmt.datalake.store import DataLakeStoreAccountManagementClient
+from azure.cli.core.commands.arm import parse_resource_id
+from azure.cli.core._util import CLIError
+
+# Helpers
+def _get_resource_group_from_account_name(client, account_name):
+    """
+    Fetch resource group from vault name
+    :param str vault_name: name of the key vault
+    :return: resource group name or None
+    :rtype: str
+    """
+    for acct in client.list():
+        id_comps = parse_resource_id(acct.id)
+        if id_comps['name'] == account_name:
+            return id_comps['resource_group']
+    return None
```

Comment:
How many commands use this?  Rather than check for None in your commands, it might make more sense to simply throw the error here (as you do in validate_resource_group_name)